src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -243,14 +243,15 @@
 
       var googleApiClient = [
         'gl-node/' + process.versions.node,
-        @if context.isGcloud
-          # TODO: might need to allow customizing the path to the package.json.
-          'gccl/' + require('../../package.json').version,
-        @end
-        CODE_GEN_NAME_VERSION,
+        CODE_GEN_NAME_VERSION
+      ];
+      if (opts.libName && opts.libVersion) {
```

Comment:
Hmm, Sorry I missed replying.
I think the current code is okay, here's why.

I believe `libName` and `libVersion` comes only from google-cloud-node hand-written libraries. Under that assumption, I believe `libVersion` needs to exist, otherwise something is wrong on GCN part (or non-GCN code wrongly specifies the code). Either way, it's harmless to silently ignore the case that libVersion is missing.